pkgs/build-support/cc-wrapper/add-clang-cc-cflags-before.sh

```diff
@@ -6,6 +6,6 @@ for p in "${params[@]}"; do
     esac
 done
 
-if $needsTarget; then
+if $needsTarget && ! [[ "${0/*cpp/cpp}" == "cpp" ]]; then
```

Comment:
```suggestion
if $needsTarget && [[ $0 != *cpp ]]; then
```

can just use the pattern matching available with `[[ ... ]]`

Reply:
Done